src/mmw/js/src/modeling/templates/controls/precipitation.html

```diff
@@ -1,5 +1,4 @@
-<select class="selectpicker" data-style="btn btn-sm btn-primary" data-size="3"> <!-- Weather Tools -->
-    <option value="24_hours">24 Hour</option>
-    <option value="historical">Historical</option>
-    <option value="custom">Custom</option>
-</select> <!-- End Weather Tools -->
+<label>
+    Rain Level
+    <input type="range" min="0" max="100" step="1" />
```

Comment:
That's a hell of a rainstorm.  Can we change the steps to be from, say, 1 to 10 but with fractional output?  Ie, 0.1, 0.2....10.0